config/crds/migration.openshift.io_directvolumemigrationprogresses.yaml

```diff
@@ -180,6 +183,8 @@ spec:
               description: PodPhase is a label for the condition of a pod at the current
                 time.
               type: string
+            runtime:
```

Comment:
`runtime` has a conflicting meaning in this context. I'd prefer `duration` or `elapsedTime` instead

Reply:
+1, `duration` can be an overloaded term as well, `elapsedTime`, +1

Reply:
+1 on `elapsedTime`

Reply:
updated